tests/test_agents/test_langchain_output_handler.py

```
@@ -110,3 +110,66 @@ def test_agent_take_next_step(agent, run_kwargs):
+@pytest.mark.fat
+def test_anthropic_output_handler_error_recovery():
+    """Test that Anthropic models handle output handler errors correctly.
+
+    This tests the fix for the "non-consecutive system messages" error that occurs
+    when using Anthropic models with force_output_handler=True. The error was caused
+    by passing error feedback as system messages instead of human messages.
+
+    See: https://github.com/anthropics/anthropic-sdk-python/issues/XXX
+    """
+    try:
+        from langchain_anthropic import ChatAnthropic
```

Flake8 (7.3.0)

[error] 'langchain_anthropic.ChatAnthropic' imported but unused

(F401)